### PR TITLE
Classes aggregation

### PR DESCRIPTION
**Pull request by RotaruDan**

### CODE REVIEW

**Comment by RotaruDan** [comment]
This pr is for this branch: https://github.com/e-ucm/rage-analytics-realtime/pull/37

File: common/src/main/java/es/eucm/rage/realtime/simple/filters/HasActivityIdFilter.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+			if (!(activityIdObject instanceof String)) {
+				if (LOG) {
+					LOGGER.info(TopologyBuilder.ACTIVITY_ID_KEY
+							+ " is a String, is: "
+							+ activityIdObject.getClass());
+				}
+				return false;
+			}
+
+			Object activityNameObject = trace
```
Comment (Synpheros) [inline common/src/main/java/es/eucm/rage/realtime/simple/filters/HasActivityIdFilter.java]: If it's a "HasActivityIdFilter" why also must have a name, a classId and a child activityId?
Change the name or move the functionality.

File: glp-example/src/main/java/es/eucm/rage/realtime/simple/filters/IsBubbledTrace.java [hunk 2]
```diff
@@ -57,7 +57,7 @@ public boolean isKeep(TridentTuple objects) {
 					.get(TopologyBuilder.CHILD_ACTIVITY_ID_KEY);
 
 			if (childActivityObject != null) {
-				// Is has been bubbled it will have a CHILD_ACTIVITY_ID_KEY
+				// If has been bubbled it will have a CHILD_ACTIVITY_ID_KEY
```
Comment (Synpheros) [inline glp-example/src/main/java/es/eucm/rage/realtime/simple/filters/IsBubbledTrace.java]: Thanks

**Comment by RotaruDan** [comment]
Cloring and reopening to master

**Pull request closed without merging**